standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1826,6 +1826,13 @@ public enum ConfVars {
             new StringSetValidator("simple", "jwt"),
         "Property-maps servlet authentication method (simple or jwt)."
     ),
+    ICEBERG_CATALOG_SERVLET_FACTORY("hive.metastore.catalog.servlet.factory",
+            "hive.metastore.catalog.servlet.factory",
+            "org.apache.iceberg.rest.HMSCatalogServer",
```

Comment:
config is called `factory`, should we rename `HMSCatalogServer` to `HMSCatalogFactory`?